assets/src/edit-story/components/canvas/layout.js

```diff
@@ -47,7 +47,7 @@ export const Z_INDEX = {
   EDIT: 3,
 };
 
-const MENU_HEIGHT = 48;
+const MENU_HEIGHT = 52;
```

Comment:
This (random magic number) will all be done [properly](https://github.com/google/web-stories-wp/pull/6236/files#diff-f634fcb15355d7101d454a9ad3b2aa06ceb1ce6bd5ebd0f4e8dcf55c9ded3441R59) in the zoom PR in #6236 , so ignore for this one.